mutagen/wavpack.py

```diff
@@ -90,10 +91,13 @@ def __init__(self, fileobj):
         self.version = header.version
         self.channels = bool(header.flags & 4) or 2
         self.sample_rate = RATES[(header.flags >> 23) & 0xF]
+        bytes_per_sample = (header.flags & 1) << 1 | header.flags & 2
```

Comment:
Are you sure? It seems to me `(header.flags & 0x3) + 1` is what we want here according to http://www.wavpack.com/WavPack5FileFormat.pdf